### PR TITLE
New Elasticsearch Cheat Sheet

### PR DESCRIPTION
**Pull request by varundey**
https://duck.co/ia/view/elasticsearch_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@varundey Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@varundey thanks for making this!

In order for this to move ahead I need you to make an IA Page and add a link to the IA Page to your PR Description above (currently blank)

File: share/goodie/cheat_sheets/json/elasticsearch.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+	"id": "elasticsearch_cheat_sheet",
+	"name": "Elasticsearch",
+	"description": "An overview of commonly used Elasticsearch API commands",
+	"metadata": {
+		"sourceName": "Elastic",
+		"sourceUrl": "https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html"
+	},
+	"aliases": [
+		"elastic", "es", "elastic search", "elasticsearch kibana", "elasticsearch marvel", "elasticsearch sense", "es kibana", "es marvel", "es sense", "kibana", "marvel", "sense"
```
Comment (varundey) [inline share/goodie/cheat_sheets/json/elasticsearch.json]: Done. Please check and let me know if any changes are yet to be made. :+1:

File: share/goodie/cheat_sheets/json/elasticsearch.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
+{
+	"id": "elasticsearch_cheat_sheet",
+	"name": "Elasticsearch",
+	"description": "An overview of commonly used Elasticsearch API commands",
+	"metadata": {
+		"sourceName": "Elastic",
+		"sourceUrl": "https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html"
+	},
+	"aliases": [
+		"elastic", "es", "elastic search", "elk"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/elasticsearch.json]: We can probably just have `elastic search`.

**Comment by laurenancona** [comment]
I'm stoked to use this, @varundey, thank you!
 
We may want to consider removing or swapping out the `Elasticsearch Rivers Basics` section, as Rivers were [deprecated](https://www.elastic.co/guide/en/elasticsearch/rivers/current/index.html) in v1.5 & are not present in current release (v2.2) in lieu of using [Logstash](https://www.elastic.co/guide/en/logstash/current/index.html) + [plugins](https://www.elastic.co/guide/en/logstash/current/working-with-plugins.html).

https://www.elastic.co/blog/deprecating-rivers

**Comment by varundey** [comment]
Thanks a lot @laurenancona for pointing that out. I have removed it :smile:

**Comment by varundey** [comment]
Hi @moollaza .
It's been long since I got a reply for this PR. Please let me know the status.
Thanks

**Comment by alohaas** [comment]
@varundey please apologize for our tardiness. It looks like your Cheat Sheet only needs a little more review before it can be merged. I'll ping @moollaza.

You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Elastic Search and data filtering tools.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Elastic Search or data filtering

OR

- Test live programming or data-related Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by moollaza** [comment]
@varundey Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@varundey this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**